instrumentation/jetty-httpclient/jetty-httpclient-9.2/javaagent/build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group = "org.eclipse.jetty"
+    module = 'jetty-client'
+    versions = "[9.2,9.4.+)"
+  }
+}
+
+
+//Jetty client 9.2 is the best starting point, HttpClient.send() is stable there
+def jettyVers_base9 = '9.2.0.v20140526'
+
+
+dependencies {
+  implementation project(':instrumentation:jetty-httpclient:jetty-httpclient-9.2:library')
+
+  library "org.eclipse.jetty:jetty-client:${jettyVers_base9}"
+  latestDepTestLibrary "org.eclipse.jetty:jetty-client:9.+"
+
+  testImplementation project(':instrumentation:jetty-httpclient:jetty-httpclient-9.2:testing')
+  testImplementation("org.eclipse.jetty:jetty-server:${jettyVers_base9}") {
```

Comment:
Does the test use jetty server?

Reply:
Just tested without that jetter-server reference, it seemed to work